### DISCUSSION TITLE
127.0.0.1 拒绝访问  AList进不去

### DISCUSSION THREAD

**Discussion by xiaoenfeng** [AlistGo/alist#3013] [Q&A]
无法访问此网站127.0.0.1 拒绝了我们的连接请求。
AList进不去 o(╥﹏╥)o
求大佬们有好的办法么

**Comment by xiaoenfeng** [comment]
![image](https://user-images.githubusercontent.com/121928656/211952963-25805073-854f-46ad-b69d-8729041a6c86.png)

**Comment by NovaShen555** [comment]
看看service端日志

**Reply by Jacobzhang-Riny** [reply]
具体在哪里看？

**Comment by leblos** [comment]
我也进不去了，有什么办法吗

**Comment by Dashan-IZ** [comment]
这个是很基础的问题，服务器安全策略开放对应端口，还有服务器要开放对于端口

**Reply by crazypeng2828** [reply]
如何开呢，请教一下。

**Reply by NovaShen555** [reply]
本地访问跟端口没关系吧？
